h2o-py/h2o/frame.py

```diff
@@ -2344,6 +2344,27 @@ def isax(self, num_words, max_cardinality, optimize_card=False, **kwargs):
         if max_cardinality <= 0: raise H2OValueError("max_cardinality must be greater than 0")
         return H2OFrame._expr(expr=ExprNode("isax", self, num_words, max_cardinality, optimize_card))
 
+    def pivot(self, index, column, value):
+        """
+        Pivot the frame designated by the three columns: index, column, and value
+        Index is a column that will be the row label
+        Column is a column that contains categorical levels which will each be turned into a column
+        Value is a column associated with an index and column
+
+        :param index:
+        :param column:
+        :param value:
+        :return:
+        """
+        if index not in self.names:
```

Comment:
`self.names` creates a copy of the internal list of names -- so better not call this method more than once